eZ/Publish/Core/Persistence/Cache/ContentTypeHandler.php

```diff
@@ -251,7 +254,10 @@ public function unlink( $groupId, $typeId, $status )
         $return = $this->persistenceHandler->contentTypeHandler()->unlink( $groupId, $typeId, $status );
 
         if ( $status === Type::STATUS_DEFINED )
+        {
             $this->cache->clear( 'contentType', $typeId );
+            $this->cache->clear( 'fieldMap' );
```

Comment:
No, you are right -- it does not need clearing here. Fixed in https://github.com/ezsystems/ezpublish-kernel/commit/410eff351c27554c74714395ab34d22a52734c6b.